docroot/modules/custom/mass_map/src/MapLocationFetcher.php

```diff
@@ -249,21 +249,23 @@ public function getLocations(array $nids) {
         // Get hours from the referenced contact info node.
         if (!$contact_information_entity->field_ref_hours->isEmpty()) {
           $hours_paragraph = $contact_information_entity->field_ref_hours->entity;
-          $hours = $hours_paragraph->field_hours_structured->view('default');
-          if ($hours) {
-            $hours = \Drupal::service('renderer')->render($hours);
-            $hours = str_replace('<p>', '', $hours);
-            $hours = str_replace('</p>', '|', $hours);
-            $hours = array_map('trim', explode('|', $hours));
-            foreach ($hours as $hour) {
-              if (!$hour) {
-                continue;
+          if (!is_null($hours_paragraph) &&   !is_null($hours_paragraph->field_hours_structured->entity)) {
```

Comment:
Didn't notice during my review @vordude but if we still need this check, we should eliminate that extra space after `&&`